lib/vm_service_wrapper.dart

```diff
@@ -0,0 +1,268 @@
+import 'dart:async';
+import 'package:vm_service_lib/vm_service_lib.dart';
+
+class VmServiceWrapper implements VmService {
+  VmServiceWrapper(this._vmService);
+
+  VmServiceWrapper.fromNewVmService(
+      Stream<dynamic> /*String|List<int>*/ inStream,
+      void writeMessage(String message),
+      {Log log,
+      DisposeHandler disposeHandler}) {
```

Comment:
add a trailing comma after 
`disposeHandler` to get nicer formatting of this constructor 
e.g.
`DisposeHandler disposeHandler,
})`

Reply:
done